File: NetworkPass-master/NetWorkPassServer/src/NetWorkPassServer.Application/Dtos/BranchDto.cs
using NetWorkPassServer.Domain.Branches;


namespace NetWorkPassServer.Application.Dtos;
public sealed record BranchDto(
    Guid Id,

    string Name,

    string City,

    string District,

    string FullAddress,

    string PhoneNumber1,

    string? PhoneNumber2,

    string Email,

    string? WanIp,

    string? Subnet,

    string? Gateway,

    string? DnsServer,

    BranchType Type,

    BranchStatus Status,

    int TotalDeviceCount,

    int OnlineDeviceCount,

    int OfflineDeviceCount,

    int WarningDeviceCount,

    int AlertCount,

    DateTime? LastSeenAt,

    bool IsMonitoringEnabled,

    bool IsActive,

    DateTime CreationTime,

    DateTime? LastModificationTime
);

File: NetworkPass-master/NetWorkPassClient/apps/Admin/src/Branch/data-access/models/BranchDto.ts


export interface BranchDto {

id: string;

code: string;

name: string;

city: string;

type: string;

status: string;

totalDeviceCount: number;

onlineDeviceCount: number;

degradedDeviceCount: number;

offlineDeviceCount: number;

alertCount: number;

healthScore: number;

isActive: boolean;

lastSeenAt: string | null;
}

